mmdet/datasets/pipelines/transforms.py

```diff
@@ -854,9 +854,14 @@ def __call__(self, results):
 
         h, w, c = img.shape
         ratio = random.uniform(self.min_ratio, self.max_ratio)
-        expand_img = np.full((int(h * ratio), int(w * ratio), c),
-                             self.mean,
-                             dtype=img.dtype)
+        if self.mean[0] == self.mean[1] and self.mean[1] == self.mean[2]:
```

Comment:
- We can leave a comment here to tell users why.
- The processed image does not necessarily have 3 channels, we can use `if np.all(self.mean == self.mean[0])` instead.

Reply:
Done